parachain/pallets/ethereum-beacon-client/src/benchmarking/data.rs

```diff
@@ -8,14 +8,10 @@ use snowbridge_beacon_primitives::{
 };
 use sp_core::U256;
 use sp_std::vec;
-use std::time::{SystemTime, UNIX_EPOCH};
 
 pub fn initial_sync<SyncCommitteeSize: Get<u32>, ProofSize: Get<u32>>(
 ) -> InitialSync<SyncCommitteeSize, ProofSize> {
-	let time_now = SystemTime::now()
-		.duration_since(UNIX_EPOCH)
-		.expect("Time went backwards")
-		.as_secs();
+	let time_now = 1675679352; //2023.2.6
```

Comment:
would assume provide some arbitrary mock data for `time_now` should be fine

Reply:
Yes! 😄